packages/grafana-data/src/themes/createShape.ts

```diff
@@ -7,8 +7,29 @@ export interface ThemeShape {
   radius: Radii;
 }
 
-interface Radii {
+export interface Radii {
+  /**
+   * Use for most things (inputs, buttons, cards, panels, etc)
+   * Same as `md`
+   */
   default: string;
+  /**
+   * Use for most things (inputs, buttons, cards, panels, etc)
+   * Same as `default`
+   */
+  md: string;
+  /**
+   * Use for smaller things like chips, tags and badges
+   * Same as `default`
```

Comment:
think "Same as `default`" is incorrect here. copy pasta?

Reply:
this one is still incorrect - i think you changed the `md` one instead of the `sm` one